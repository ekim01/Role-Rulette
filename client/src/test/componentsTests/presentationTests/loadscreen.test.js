const filename = "loadscreen.js";

import React from "react";
import { create } from "react-test-renderer";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import Loadscreen from "../../../components/presentation/loadscreen";

describe(filename + " LoadScreen component", () => {
  test("Matches the snapshot", () => {
    const router = create(<Loadscreen />);
    expect(router.toJSON()).toMatchSnapshot();
  });
});
