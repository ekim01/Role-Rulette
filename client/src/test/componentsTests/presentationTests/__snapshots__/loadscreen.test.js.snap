// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadscreen.js LoadScreen component Matches the snapshot 1`] = `
<div
  className="overlay"
>
  <i
    className="fa fa-refresh fa-spin"
  />
  <span>
    <h1 />
  </span>
</div>
`;
